content/docs/dvclive/api-reference/live/log_image.md

```diff
@@ -0,0 +1,77 @@
+# Live.log_image()
+
+Saves the given image `val` to the output file `name`.
+
+```py
+def log_image(name: str, val):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+import numpy as np
+img_numpy = np.ones((500, 500), np.uint8) * 255
+live.log_image("numpy.png", img_numpy)
+
+from PIL import Image
+img_pil = Image.new("RGB", (500, 500), (250, 250, 250))
+live.log_image("pil.png", img_pil)
+```
+
+## Description
+
+Supported values for `val` are:
+
+- A valid numpy array (convertible to image via
+  [PIL.Image.fromarray](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.fromarray))
+- A `PIL.Image` instance.
+
+The images will be saved in `{path}/images/{name}`:
+
+```dvc
+$ tree
+└── dvclive
+    └── images
+        ├── numpy.png
+        └── pil.png
+```
+
+💡 Images can be visualized with `dvc plots`.
+
+### Step updates
+
+The first `step` update (with `Live.next_step()` or `Live.set_step()`) will move
+the saved file from `{path}/images/{name}` to `{path}/images/{step}/{name}`.
```

Comment:
> First time path is referenced. Unclear where this comes from

Another option is to say `{dvclive_path}` or skip that part of the path in examples e.g. `.../images/{name}`